frontend/src/app/components/StreamlitDialog/SettingsDialog.tsx

```diff
@@ -74,14 +74,14 @@ export class SettingsDialog extends PureComponent<Props, UserSettings> {
 
   private renderThemeCreatorButton = (): ReactElement | false =>
     this.props.developerMode && (
-      <div>
+      <StyledButtonContainer data-testid="edit-theme">
```

Comment:
Just added this `data-testid` to make it a little easier to get to the Edit active theme dialog -> impacts the FE snapshot below